README.md

```diff
@@ -12,16 +12,17 @@ Please refer to the public [documentation](https://alfresco-content-app.netlify.
 ## Compatibility
 
 | ACA   | ADF           | ACS       | Node | Angular |
-|-------|---------------|-----------|------|---------|
-| 5.1.x | 7.0.0-alpha.3 | 23.3      | 18.x | 15.x    |
-| 5.0.x | 7.0.0-alpha.2 | 23.3      | 18.x | 15.x    |
-| 4.4.x | 6.7           | 23.2      | 18.x | 14.x    |
-| 4.3.x | 6.4           | 23.1      | 18.x | 14.x    |
-| 4.2.x | 6.3           | 23.1.0-M4 | 18.x | 14.x    |
-| 4.1.x | 6.2           | 7.4       | 18.x | 14.x    |
-| 4.0.x | 6.1           | 7.4       | 14.x | 14.x    |
-| 3.1.x | 5.1           | 7.3       |      |         |
-| 3.0.x | 5.0           | 7.3       |      |         |
+|-------|---------------|-----------|------|--------|
+| 5.2.x | 7.0.0-alpha.6 | 23.3      | 18.x | 16.x   |
```

Comment:
We released ACS 23.4 now so this will be compatible version

Reply:
Updated